src/Compilers/Core/Portable/Operations/OperationVisitor.cs

```diff
@@ -534,7 +534,7 @@ public virtual TResult VisitUsingStatement(IUsingStatement operation, TArgument
         {
             return DefaultVisit(operation, argument);
         }
-        
+
         public virtual TResult VisitFixedStatement(IFixedStatement operation, TArgument argument)
```

Comment:
For some reason CodeFlow shows that as two added lines, must be a problem with the tool.